.github/workflows/cd.yml

```diff
@@ -0,0 +1,29 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+#  pull_request:
+
+jobs:
+  push-executors:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get previous tag
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: ""
+      - name: push to Hubble
+        run: |
+          bash ./.github/workflows/scripts/push.sh .
+        env:
+          token: ${{ secrets.GH_TOKEN }}
+          GIT_TAG: ${{ steps.previoustag.outputs.tag }}
```

Comment:
Instead of maintaining scripts in each repo, please use the GA https://github.com/jina-ai/action-hubble-push. 
You can use the `github workflow template` as described at https://docs.google.com/document/d/1tTc5oVFqAWQlFjuV5NPS-JbknV8_2Ooe_7EF3jpZQK0/edit#heading=h.jb8g4buzk04m